src/thinblock.h

```diff
@@ -18,6 +18,8 @@
 class CDataStream;
 class CNode;
 
+extern std::string formatInfoUnit(double value);
```

Comment:
add a docstring comment describing the operation of this function.  See https://github.com/BitcoinUnlimited/BitcoinUnlimited/blob/release/doc/developer-notes.md#doxygen-comments